.travis.yml

```diff
@@ -1,24 +1,11 @@
 language: php
 
-before_install:
-    - sudo apt-get update -qq
-    - sudo apt-get install re2c
-
 before_script:
-    - ./travis/compile-php7.sh
-    - PATH=`pwd`:$PATH composer self-update
-    # using PHP 7 means stuff depending on ">=5.3.3" etc. breaks
-    # so we make composer ignore it
-    - PATH=`pwd`:$PATH composer install --dev --prefer-source --ignore-platform-reqs
+    - composer install --prefer-source --ignore-platform-reqs
 
 script:
-    # use PHP 7 for the actual tests
-    - PATH=`pwd`:$PATH make test
+    - make test
 
 php:
   - 5.6 # lol
```

Comment:
YOLO only needs to run on the latest and greatest, so this line should be removed.

BC is for the swagless.